nuxt.config.ts

```diff
@@ -16,4 +16,7 @@ export default defineNuxtConfig({
   },
   devtools: { enabled: true },
   modules: ['@nuxt/ui', 'nuxt-gtag'],
+  gtag: {
+    id: 'G-GMQPKYN09N'
+  }
```

Comment:
Gtag can be injected in Netlify <img width="981" alt="image" src="https://github.com/likecoin/citywalk-until-campaign/assets/17264716/69d2c547-3e8e-46b4-983f-2c7fc6b62903">

Reply:
removed commit